kafka/config.go

```diff
@@ -183,6 +183,14 @@ func (m ConfigMap) convert() (cConf *C.rd_kafka_conf_t, err error) {
 // If the key is not found defval is returned.
 // If the key is found but the type is mismatched an error is returned.
 func (m ConfigMap) get(key string, defval ConfigValue) (ConfigValue, error) {
+	if strings.HasPrefix(key, "{topic}.") {
```

Comment:
ahh, ok - sounds good. FWIW I'd be in favor of depreciating then removing this, but argument is good the other way, and not a big deal.